cloud-sql/mysql/mysql/server.js

```diff
@@ -122,23 +122,28 @@ const ensureSchema = async pool => {
   console.log("Ensured that table 'votes' exists");
 };
 
+const createPoolAndEnsureSchema = async () =>
+  await createPool()
```

Comment:
Since this is now inside an async function, we can simplify: 

```
try {
  const pool = await createPool();
  await ensureSchema(pool);
  return pool;
} catch (err) {
  logger.error(err);
  throw err;
}
```